### ISSUE TITLE
Integration tests sometimes hang randomly

### ISSUE THREAD

**Issue by didimitrie** [specklesystems/Core#34]
### Expected vs. Actual Behavior

Integration tests should run, and fail or pass normally. What happens one in three times is that they hang for ever and CircleCI times them out at one point. 

### Reproduction Steps & System Config (win, osx, web, etc.)

See Core's [pipeline](https://app.circleci.com/pipelines/github/specklesystems/Core/60/workflows/12359346-9fa6-416b-bcce-5dab36c4f644/jobs/117). If manually run again, they pass.  The log is not very informative:

```
Starting test execution, please wait...

A total of 1 test files matched the specified pattern.

Too long with no output (exceeded 10m0s): context deadline exceeded
```

**Comment by didimitrie** [comment]
Note: https://github.com/microsoft/vstest/issues/2080

**Comment by didimitrie** [comment]
closed in #49
